### DISCUSSION TITLE
which sensors do what?

### DISCUSSION THREAD

**Discussion by crankycowboy** [moralmunky/Home-Assistant-Mail-And-Packages#769] [General]
Is there anywhere that explains what each sensor does? I've enabled USPS, Fedex and UPS but have 25 entities. some are self explanatory, others I can't figure out. I was wanting to use a custom card for this integration, but I'm having trouble nailing down which sensors I want to use, and without any packages/mail heading my way it's hard to figure it out. Is this listed somewhere?

**Comment by firstof9** [comment]
Which sensors seem unclear?

**Reply by crankycowboy** [reply]
So it appears that most providers have a packages, exception, delivered, delivering, in transit, but then also, what sensor is adding up total packages? Basically, I want to create a card that shows incoming packages/mail for usps, amazon, ups and fedex, and then also show delivered for the same services (similar to the attached image). I've also attached a list of all my entities.
![sensors1](https://user-images.githubusercontent.com/2176738/208326581-b21081f2-1b8c-41ad-b4c9-0d9a0a78bac1.png)
![sensor2](https://user-images.githubusercontent.com/2176738/208326582-3037880b-12c8-4c64-bf96-eb280af838d1.png)
![400307f18e357b3d21263613b03328b3e29ed6ae](https://user-images.githubusercontent.com/2176738/208326583-fa7ca15a-a174-4688-8161-126b66fea5ad.png)

**Comment by firstof9** [comment]
`Packages Delivered` is the total delivered and `Packages In Transit` is the total being delivered today.
Anything that doesn't have a carrier specific name would be the total sensor.

**Reply by crankycowboy** [reply]
Thank you again, so is "delivering" the same as "in transit"?  And with USPS where is just says mail usps mail, what is that if not in transit or delivered?  Sorry, I'm just trying to understand the status of the various entities

**Reply by firstof9** [reply]
Correct, "delivering" is "in transit" for each carrier.
USPS Mail is how many pieces of mail you're getting for the day, non-parcel.

**Reply by crankycowboy** [reply]
Thanks again first0f9. you have the patience of a saint to put up with me. I appreciate your time!

**Reply by firstof9** [reply]
No problem.
